### ISSUE TITLE
allow customization of food filters and priorities

### ISSUE THREAD

**Issue by jsb** [jsb/Gourmet#4]

**Comment by sadfx** [comment]
Being unable to (e.g.) eat Lvl 0 food at Lvl 6 falls under this issue eh? Not good enough to fix this myself :<

**Comment by jsb** [comment]
@sadfx: What's your intention exactly? You want to prevent Gourmet to select level 0 food for eating when your character is level 6? If so then, yes, this is a possible application of what I intend to implement for this feature.

**Comment by sadfx** [comment]
Yes, it'd be very nice to be able to eat lower lvl food should same lvl range food not exist :) Something else I noticed is that because this requires 'BAG_UPDATE' to work, it doesn't work when you first log in (might just be at lvl 1 I haven't looked at the code too much :P), a problem that isn't a real problem but perhaps an unnecessary problem :P 

Some other suggestions: best bandage, best mana/hp potion, healthstone and other health recovery items such as the Night Dragon's Breath (perhaps they should be put in the same command as hppot)

**Comment by jsb** [comment]
Yes, this is what I have in mind. Thank you for your filter / priority suggestions, they have been noted.
